### PR TITLE
Added Droidcon Lisbon 2024

### PR DESCRIPTION
**Pull request by Kardelio**

### CODE REVIEW

File: _conferences/droidcon-lisbon-2024.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+name: "Droidcon"
+website: https://www.lisbon.droidcon.com/
+location: Lisbon, Portugal
+
+date_start: 2024-09-05
+date_end:   2024-09-06
+
+cfp:
+  start: 2024-04-01
```
Comment (cketti) [inline _conferences/droidcon-lisbon-2024.md]: For me the sessionize page shows that the call for speakers opened 2024-02-01

image

**Review by cketti — CHANGES_REQUESTED** [review]

File: _conferences/droidcon-lisbon-2024.md (hunk 1 above)
Comment (Kardelio) [reply _conferences/droidcon-lisbon-2024.md]: Oh crap sorry i messed up the date there my bad fixing now

**Comment by Kardelio** [comment]
@cketti fixed issue in new commit

**Review by cketti — APPROVED** [review]

**Comment by cketti** [comment]
Thanks :+1:

**Pull request merged**